apps/webapp/app/services/endpointApi.server.ts

```diff
@@ -240,6 +241,36 @@ export class EndpointApi {
     return HttpSourceResponseSchema.parse(anyBody);
   }
 
+  async deliverHttpEndpointRequestForResponse(options: {
```

Comment:
This is used when the user wants an immediate `Response` to their `Request` (by implementing `responseWith` on their HTTP endpoint.